tests/unit_tests/simpll/SimpLLTest.cpp

```diff
@@ -1,6 +1,36 @@
+#include "Logger.h"
+
+#include <cerrno>
 #include <gtest/gtest.h>
+#include <ostream>
+
+/// Sets logging verbosity based on `SIMPLL_VERBOSITY` env. var.
+/// Returns true if successful, otherwise returns false.
+int trySetVerbosity() {
+    const char *verbosity = std::getenv("SIMPLL_VERBOSITY");
+    if (!verbosity)
+        return true;
+    int level;
```

Comment:
Just to make the code absolutely clear define its value. Perhaps you can use `int level{}`.